### ISSUE TITLE
Finish 1.3-causal-relation-presence-bert-for-sequence-classification

### ISSUE THREAD

**Issue by svenvanderburg** [SSIML2021/SSIML2021#27]
Finish the experiment 1.3-causal-relation-presence-bert-for-sequence-classification.
- [ ] Use fixed Kfold splits across all experiments. (use random seed, see #16 and #22)
- [ ] I suggest to use 5-fold instead of 10-fold to speed up the experiments. I think it wouldn't hurt performance that much.
- [ ] IMO you can get rid of all the classification reports and confusion matrices for the training data, it doesn't add much information.
- [ ] Train for more than 3 epochs. It seems like the train and val loss are still dropping, so the model has not fully converged yet. You can find the right number of epochs manually (I would suggest to do a few experiments on only 1 fold to save time), or you can use early stopping.
- [ ] You are using the test set as validation set, this is not a good practice. So, either train BERT without a validation set. Or split your training set further into a train and validation set. NB @Smu-Tan I see you sometimes call the test set 'evaluation set', with 'test set' I mean the set on which you do the final evaluation, the validation set is used to tune hyperparameters or monitor training progress.
